common/persistence/persistenceMetricClients.go

```diff
@@ -898,6 +898,36 @@ func (p *executionPersistenceClient) AppendRawHistoryNodes(
 	return resp, err
 }
 
+// ParseHistoryBranchInfo parses the history branch for branch information
+func (p *executionPersistenceClient) ParseHistoryBranchInfo(
+	ctx context.Context,
+	request *ParseHistoryBranchInfoRequest,
+) (*ParseHistoryBranchInfoResponse, error) {
+
+	branchInfo, err := ParseHistoryBranchToken(request.BranchToken)
```

Comment:
I don't follow. The call should be forward to `p.persistence`? Same Q for all three client impl.

Reply:
darn... must have been a bizarre mishap (my mistake) when I rebased. This is not the code I intended! Let me fix this